src/SerializeAST.cc

```diff
@@ -841,48 +844,74 @@ sos::Object drafter::WrapBlueprint(const Blueprint& blueprint, const ASTType ast
         }
 
     }
-#endif
+}
 
-    if (astType == RefractASTType) {
-        refract::IElement *element = NULL;
+sos::Object BlueprintAsRefractAST(const Blueprint& blueprint)
```

Comment:
And this shouldn't be under `WITH_REFRACT` since that variable is for MSON Refract.